CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+# Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment where everyone gets a chance to learn, we as
+contributors and maintainers pledge to making participation in our project, Oppia and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Being available to help other other contributors.
+* Being respectful of differing viewpoints and experiences.
+* Focusing on teaching-oriented communication so that each contributor gets a chance to learn and grow.
+* Focusing on what is best for the Oppia community.
+* Gracefully accepting constructive criticism.
+* Showing empathy towards other community members.
+* Using welcoming and inclusive language for all.
```

Comment:
I don't think you need "for all" here, it's already covered by "inclusive".

Reply:
Yes, I agree. Removed. Thanks!